service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java

```diff
@@ -253,6 +253,9 @@ public static void executeOnEachServiceRegistry(Consumer<ServiceRegistry> action
 
   public static void addExtraServiceRegistry(ServiceRegistry serviceRegistry) {
     Objects.requireNonNull(serviceRegistry);
+    if (EXTRA_SERVICE_REGISTRIES.containsKey(serviceRegistry.getName())) {
+      throw new IllegalArgumentException("Register name duplicated!");
```

Comment:
reigstriy name {name} is duplicated, please set different names for each implementation.

Reply:
when duplicated, I can only get the new client name, cannot get old client name from EXTRA_SERVICE_REGISTRY. How to set different implmentation?